src/embedme.ts

```diff
@@ -97,13 +97,17 @@ sourceFiles.forEach((source, i) => {
 
   const sourceText = readFileSync(source, 'utf-8');
 
-  const outText = embedme(sourceText, resolvedPath, options);
+  const { outText, error } = embedme(sourceText, resolvedPath, options);
 
   if (options.verify) {
     if (sourceText !== outText) {
       errorLog(chalk => chalk.red(`Diff detected, exiting 1`));
       process.exit(1);
     }
+
+    if (error) {
```

Comment:
this should move outside of the `if (options.verify) {` block - if you're doing a regular run, or dry run etc it should still `exit(1)` if there is an error regardless of whether or not the --verify flag was passed.

Reply:
Hi @zakhenry, just made this change. I did need to modify the test fixtures slightly, let me know if it looks ok. Thanks!

Reply:
hey thanks for the quick edit, I think the fixture change has reduced some coverage however. I understand that the change was needed as I guess the original checks were failing due to it now exiting with an error code, but there is now no longer a test to cover the error messages emitted on those failing cases.

Would you like me to take a look?

Reply:
Got it. Yeah if you'd like to check it out that'd be great, not sure of the best way to test those...

Reply:
ok cool I've fixed that up in #95 , this replaces this PR as I took your commits and added one of my own.

Note this won't auto deploy to npm just yet as I will need to migrate from travis ci to github actions which I will complete separately.